doc/development/adding_operators.rst

```diff
@@ -0,0 +1,356 @@
+.. _contributing_operators:
+
+Creating new operators
+======================
+
+The following steps will help you to create custom operators, and to
+potentially add them to PennyLane.
+
+Note that in PennyLane, a circuit ansatz consisting of multiple gates is also an operator - one whose
+action is defined by specifying a representation as a combination of other operators.
+For historical reasons, you find circuit ansaetze in the ``pennylane/template/`` folder,
+while all other operations are found in ``pennylane/ops/``.
+
+The base classes to construct new operators are found in ``pennylane/operations.py``.
+
+Abstraction
+###########
+
+Operators in quantum mechanics are maps that act on vector spaces, and in differentiable quantum computing, these
+maps can depend on a set of trainable parameters. The :class:`~.Operator` class
+serves as the main abstraction of such objects, and all operators (such as gates, channels, observables)
+inherit from it.
+
+.. code-block:: python
+
+   >>> from jax import numpy as jnp
+   >>> op = qml.Rot(jnp.array(0.1), jnp.array(0.2), jnp.array(0.3), wires=["a"])
+   >>> isinstance(op, qml.operation.Operator)
+   True
```

Comment:
Here (and below as well), the `.. code-block:: python` should be removed, and the indentation removed, so that it properly renders as pycon syntax :)
```suggestion
>>> from jax import numpy as jnp
>>> op = qml.Rot(jnp.array(0.1), jnp.array(0.2), jnp.array(0.3), wires=["a"])
>>> isinstance(op, qml.operation.Operator)
True
```

Reply:
Ah I thought it's both valid. Makes sense!